src/instructlab/chat/chat.py

```diff
@@ -475,6 +476,7 @@ def chat_cli(
     """Starts a CLI-based chat with the server"""
     cert = get_ssl_cert_config(tls_client_cert, tls_client_key, tls_client_passwd)
     verify = not tls_insecure
+    logger.debug("calling OpenAI API")
```

Comment:
I'm not sure whether this is a good usage of DEBUG logs. Also, this is not calling the API, we are building the client.